### DISCUSSION TITLE
Invert Selection of Hearing Impared lines

### DISCUSSION THREAD

**Discussion by Titounet1965** [SubtitleEdit/subtitleedit#6210] [Ideas]
Hi and sorry for my poor english
If you could use the dialog box of your function 'remove hearing Impared Dialox' for a selection cache for lines.
i use it for detecting 'only caps' lines and extract 'forced' subtitles for video which dont have it.

Or maybe just one or 2 extra buttons on the dialog box 
There Is OK , Cancel and Apply
If you just could add 'Select lines" and "Select all other lines"

Thanks you

**Comment by niksedk** [comment]
You could try Edit - Modify selection;

![image](https://user-images.githubusercontent.com/1261331/188267563-156e76b3-cbd9-4c43-b275-22cc32c1d43e.png)

Also supports `contains` and `regular expressions`

**Comment by Titounet1965** [comment]
TY , i had not seen this could create and/or interactions
